docs/content/Schema/Fundamentals/Concepts.md

```diff
@@ -0,0 +1,328 @@
+---
+title: Concepts
+permalink: /schema/fundamentals/concepts
+category: Data Schema
+menuOrder: 1
+subCategory: Fundamentals
+---
+
+Cube.js borrows a lot of terminology from data science, and this document is
+intended for both newcomers and regular users to refresh their understanding.
+
+We'll use a sample e-commerce database with two tables, `orders` and
+`line_items` to illustrate the concepts throughout this page:
+
+**`orders`**
+
+| **id** | **status** | **completed_at**           | **created_at**             |
+| ------ | ---------- | -------------------------- | -------------------------- |
+| 1      | completed  | 2019-01-05 00:00:00.000000 | 2019-01-02 00:00:00.000000 |
+| 2      | shipped    | 2019-01-17 00:00:00.000000 | 2019-01-02 00:00:00.000000 |
+| 3      | completed  | 2019-01-27 00:00:00.000000 | 2019-01-02 00:00:00.000000 |
+| 4      | shipped    | 2019-01-09 00:00:00.000000 | 2019-01-02 00:00:00.000000 |
+| 5      | processing | 2019-01-29 00:00:00.000000 | 2019-01-02 00:00:00.000000 |
+
+**`line_items`**
+
+| **id** | **product_id** | **order_id** | **quantity** | **price** | **created_at**             |
+| ------ | -------------- | ------------ | ------------ | --------- | -------------------------- |
+| 1      | 31             | 1            | 1            | 275       | 2019-01-31 00:00:00.000000 |
+| 2      | 49             | 2            | 6            | 248       | 2021-01-20 00:00:00.000000 |
+| 3      | 89             | 3            | 6            | 197       | 2021-11-25 00:00:00.000000 |
+| 4      | 71             | 4            | 8            | 223       | 2019-12-23 00:00:00.000000 |
+| 5      | 64             | 5            | 5            | 75        | 2019-04-20 00:00:00.000000 |
+
+## Cubes
+
+A cube represents a dataset in Cube.js, and is conceptually similar to a [view
+in SQL][wiki-view-sql]. Cubes are typically declared in separate files with one
+cube per file. Within each cube are definitions of measures and dimensions.
```

Comment:
Would it make sense to make `measures` and `dimensions` links to the respective sections below?